### DISCUSSION TITLE
TinyMCE - enlarge to full screen

### DISCUSSION THREAD

**Discussion by mariantanase** [joomla/joomla-cms#47116] [2. Cold Features]
The first step to improve the TinyMCE editor has been done in Joomla 4.1 with the possibility of code highlighting.
Currently, the source code pop-up window of the TinyMCE editor has a fixed size.
When working with long articles with different formatting, it would be useful to have the possibility to enlarge the window in full screen.

Can be improved further ?

![source-code](https://user-images.githubusercontent.com/3976174/154457734-def45a3f-216d-4960-bb10-620a755080bb.png)

**Comment by vewert** [comment]
I just started using Joomla 4 and for the most part I'm loving TinyMCE, but agree it would be great if the editor window could be re-sized.

**Comment by mariantanase** [comment]
Is it possible to solve this problem ?
it's been almost 2 years.
Thank you
